### ISSUE TITLE
Compatibility issue in Redirect Notice screen (URL redirection from Licensing agreement screen)

### ISSUE THREAD

**Issue by ShreyaaSahu** [Path-Check/covid-safe-paths#739]
## Version

Type of phone: iPhone
OS Version: ios 13.3
Model: iPhone XR

## Steps to reproduce

1. Launch App
2. Tap on "Get Started " button
3. License Agreement Screen displays 
4. Two URL present  - in the "Term of Use" Content
5. Tap on any of them e.g.  " https://github.com/tripleblindmarket/safe-places "
6. Redirect Notice Screen will get open
7. Compatibility issues in this Screen -  content is right aligned 

![IMG-4661](https://user-images.githubusercontent.com/64080646/80707251-4e206800-8b07-11ea-8705-862dcde4506e.PNG)

![IMG-4662](https://user-images.githubusercontent.com/64080646/80707256-5082c200-8b07-11ea-9a0e-821d599487c7.PNG)

## Expected Behavior
Redirect Notice Screen - Content centre aligned

**Comment by NaziyaNazir** [comment]
Issue exists in Android 9 as well.

![Screenshot_2020-04-30-12-37-47-53_7279f4f2abc8c6febb8fd1ee45579eaa](https://user-images.githubusercontent.com/64524151/80708330-20d4b980-8b09-11ea-827e-b4c158d7423d.png)

**Comment by pauladam316** [comment]
You can assign this to me and I'll have a look at it

**Comment by pauladam316** [comment]
So I had a look into this issue and it ended up being a bit more complicated than I expected. The short version is that we can’t change that justification because we’re not generating the redirect notice. I did some digging trying to understand where those are coming from and it seems like the links in the EULA are actually google redirect links. For the life of me I can’t figure out why it was done that way, so I replaced them with direct links. That removes the redirect notices all together and solves the problem in a roundabout way. Now the other thing I noticed while working on this is that if you follow the link there is no way to return to the EULA. It also seems really silly to me to have the links in the EULA load GitHub pages inside the app, so I made a small modification so that now if you click any of the links inside the EULA, the app will open them in your phone's default browser. I think it makes a ton more sense and makes the app more usable.
